modules/spring-config/src/main/java/org/mule/runtime/config/spring/ModuleDelegatingEntityResolver.java

```diff
@@ -56,7 +63,8 @@ public InputSource resolveEntity(String publicId, String systemId) throws SAXExc
 
   private InputSource generateModuleXsd(String publicId, String systemId) {
     InputSource inputSource = null;
-    if (extensionManager.isPresent()) {
+    if (extensionManager.isPresent()
```

Comment:
Is not an error if the extensionManager is absent and the schemaResourceFactory is present? (and viceversa)